### ISSUE TITLE
Not PHP 7 compatible

### ISSUE THREAD

**Issue by wdebusschere** [DeuxHuitHuit/remote_selectbox#4]
```
Symphony Fatal Error: Class 'fieldremote_selectbox' not found
An error occurred in /Users/xxx/Sites/xxx/symphony/lib/toolkit/class.fieldmanager.php around line 582

577
578 if (!class_exists($classname)) {
579 require_once($path);
580 }
581
582 self::$_pool[$type] = new $classname;
583
584 if (self::$_pool[$type]->canShowTableColumn() && !self::$_pool[$type]->get('show_column')) {
585 self::$_pool[$type]->set('show_column', 'yes');
586 }
```

**Comment by nitriques** [comment]
What Symphony are you running ?

**Comment by wdebusschere** [comment]
Symphony 2.7.3


`Symphony Warning: Declaration of extension_remote_selectbox::update($previousVersion) should be compatible with Extension::update($previousVersion = false)
An error occurred in /Users/xxx/Sites/xxx/extensions/remote_selectbox/extension.driver.php around line 105`

**Comment by nitriques** [comment]
@wdebusschere fixed in `1.1.0`
